### DISCUSSION TITLE
Zapp App for/für Linux/Raspbian?

### DISCUSSION THREAD

**Discussion by lxne** [mediathekview/zapp#283] [Ideas]
Hi, I am not a programmer so please don't be hard on me :)
Would it be quite an effort to make the Zapp App run on Linux/Raspbian?
It would be simply awesome to be able to use Zapp on a raspberry pi running raspbian.

Best wishes and happy easter,
Alex

**Comment by cemrich** [comment]
Hi Alex,

as far as I know there is no straighforward way of running zapp on the raspberry pi. There is however an experimental version of MediathekView for the pi (https://mediathekview.de/download/). It uses the same list of shows as Zapp.

A very nice way of watching TV on the pi is [Kodi ](https://kodi.tv/). It is easy to set up - using [libre elec](https://libreelec.tv/) for installation -, and plays nice with its [companion app Kore](https://play.google.com/store/apps/details?id=org.xbmc.kore&gl=US). With this setup you can search the Mediathek on your Android phone and then share live streams or shows to Kore, which then plays it on the pi (which may be connected to your TV). I am using this setup for quire a while now and I am very happy :)

Best Regards,
Christine

**Comment by lxne** [comment]
Thank you Christine for your very informative answer.
I had something else in mind regarding a magic mirror.
But maybe Kodi might do the trick.

Btw. Zapp 7.02 is doing great! Love it.

Best wishes,
Alex


Am Sa., 16. Apr. 2022 um 20:20 Uhr schrieb cemrich ***@***.***
